### ISSUE TITLE
Cloud Firestore

### ISSUE THREAD

**Issue by benkhachouch** [PhantomAppDevelopment/firebase-as3#22]
Hello!
Have you a AS3  plan for Cloud Firestore?
Thanks!

**Comment by agentphantom** [comment]
Yes, I will do some research and tests to see how it works with AS3. The good thing is that it has a REST API so it should be similar to the previous guides.

**Comment by jasonJamEther** [comment]
Have you looked into firestore? I am able to connect an pull data but my stream doesn't seem to pull changes.

**Comment by agentphantom** [comment]
I did took a look at some examples but I will wait until it comes out of beta.

**Comment by leossmith** [comment]
Hi there. Any updates on this? Please dont wait for it to come out of beta. Google always keeps their products to a constant beta state until they deprecated the previous product they had :P
